src/lib/components/core/header/Header.svelte

```diff
@@ -311,6 +321,7 @@ function runPython() {
         {open}
     />
     <ContextItem icon={faEnvelope} name={$_("EMAIL")} onclick={email} {open} />
+	<ContextItem icon={faComment} name={$_("FEEDBACK")} onclick={feedback} {open} />
```

Comment:
Weird, biome thinks it's fine 🤔

Reply:
unfortunately, biome currently only checks the typescript portion of .svelte files, not the html and css

Reply:
This file is formatted with spaces instead of tabs, and my editor uses tabs as configured in .editorconfig. No idea how the spaces end up here, but I've reformatted with tabs now.